src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2583,9 +2583,6 @@ because xbuild doesn't support framework reference assemblies.
 
 <Target Name="_DeployApk"
     Condition=" '$(AndroidPackageFormat)' != 'aab' ">
-  <PropertyGroup>
-    <_DeployCommand>&quot;$(AdbToolPath)adb&quot; $(AdbTarget) install -r &quot;$(ApkFileSigned)&quot;</_DeployCommand>
```

Comment:
why is this being removed? Its not mentioned in the PR description? 

I thought this and the _UnintallCommand were used by dotnet under the hood.

Reply:
I didn't see these `_DeployCommand` or `_UninstallCommand` command properties being used anywhere here or in monodroid. It looked like they were left overs from when the [`AndroidAdb` task was introduced](https://github.com/xamarin/xamarin-android/commit/1d7092d7b7382c62064eb45ccd70af5f34f62eba), and they could be safely removed?